modules/nw-metallb-configure-address-pool-vlan.adoc

```diff
@@ -0,0 +1,66 @@
+:_mod-docs-content-type: PROCEDURE
+[id="nw-metallb-configure-address-pool-vlan_{context}"]
+= Configure MetalLB address pool for VLAN
+
+As a cluster administrator, you can add address pools to your cluster to control the IP addresses on a created VLAN that MetalLB can assign to load-balancer services
+
+.Prerequisites
+
+* Install the OpenShift CLI (oc).
```

Comment:
I see an attribute for this in the common-attributes.adoc file: 

![Screenshot from 2024-10-31 16-45-59](https://github.com/user-attachments/assets/34f0c1a8-2bfe-4d3f-b2cc-2ff4ad30415e)